examples/kubernetes/dogsvscats/result-app/rg.yaml

```diff
@@ -0,0 +1,125 @@
+apiVersion: kro.run/v1alpha1
```

Comment:
Can you make one rgd for results, workers, and vote instead of 3. The ingress resource can be conditional.